pom.xml

```diff
@@ -613,7 +613,7 @@
               <arg>-language:_</arg>
               <arg>-Xlint</arg>
               <arg>-Xfuture</arg>
-              <arg>-Xfatal-warnings</arg>
+              <!-- arg>-Xfatal-warnings</arg -->
```

Comment:
I forgot about this guy. I set it to make a first build for my eclipse. I revert it.